ext/random/tests/03_randomizer/gh9415.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+GH-9415: Randomizer::getInt(0, 2**32 - 1) with Mt19937 always returns 1
+--FILE--
+<?php
+
+use Random\Randomizer;
+use Random\Engine\Mt19937;
+
+$r = new Randomizer(new Mt19937(1234));
+var_dump($r->getInt(0, 2**32 - 1));
+
+$r = new Randomizer(new Mt19937());
+var_dump($r->getInt(0, 2**32 - 1) !== 1 || $r->getInt(0, 2**32 - 1) !== $r->getInt(0, 2**32 - 1));
```

Comment:
Is this an *absolute* safe assumption?

Reply:
Yeah, might be better to have a deterministic test. An alternative might be to add some comment that this might fail in highly unlikely cases (so if it fails, nobody would have to take a closer look).